source/class/qx/ui/form/MenuButton.js

```diff
@@ -241,6 +261,7 @@ qx.Class.define("qx.ui.form.MenuButton",
     {
       switch(e.getKeyIdentifier())
       {
+        case "Space":
```

Comment:
This too. Typically Space and Enter are treated equivalently. I think maybe these enhancements and changes of behavior should be in a separate PR, though.

Reply:
Change of behavior